Portal_Perfomance_Employes/Controllers/EmployeController.cs

```diff
@@ -0,0 +1,65 @@
+﻿using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Portal_Perfomance_Employes.Data;
+using Portal_Perfomance_Employes.Models;
+
+namespace Portal_Perfomance_Employes.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class EmployeController : ControllerBase
+    {
+        private EmployesDB _employesDB = new EmployesDB();
+
+        public EmployeController()
+        {
+            _employesDB.PopularEmployesDB();
+        }
+
+
+        [HttpGet]
+        public async Task<IEnumerable<Employe>> Get() => _employesDB.employes.ToList();
```

Comment:
This works fine because we don't have a db yet- but maybe we could use `await` and `ToListAsync()` so we don't have to change it in the future?

Reply:
Could also change in the future once we need it, I would leave it up to the author to decide

Reply:
Yes, I think we will need to change it. I didn't use await or ToListAsync because it's not from the database yet and it was just a exemple.